devU-api/src/utils/swagger.utils.ts

```diff
@@ -13,7 +13,7 @@ const swaggerOptioner = {
   },
 }
 
-const jsDocOptions = { swaggerDefinition: swaggerOptioner, apis: ['./src/router/*.ts'] }
+const jsDocOptions = { swaggerDefinition: swaggerOptioner, apis: ['./src/router/*.ts', './src/course/course.router.ts'] } //Add routers to this, I think?
```

Comment:
Looks like this didn't get updated after the directory restructure. This is an artifact from when every router was in the router directory. Replacing course with * should grab all the routes. Keep the first string too since that's grabbing the paths from the index router